examples/cms/src/App.css

```diff
@@ -0,0 +1,165 @@
+    background: linear-gradient(180deg, rgba(18, 18, 18, 0) 0%, rgb(17, 17, 17) 97.8%);
+}
+
+[data-framer-theme="dark"] input[type="checkbox"]:checked {
+    background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMiIgaGVpZ2h0PSIxMiI+PHBhdGggZD0iTSAzIDYgTCA1IDggTCA5IDQiIGZpbGw9InRyYW5zcGFyZW50IiBzdHJva2Utd2lkdGg9IjEuNSIgc3Ryb2tlPSIjMmIyYjJiIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIHN0cm9rZS1kYXNoYXJyYXk9IiI+PC9wYXRoPjwvc3ZnPg==");
+}
+
+.sticky-top {
+    position: sticky;
+    top: 0;
+}
+
+.field-input {
+    width: 100%;
+    flex-shrink: 1;
+}
+
+.column-span-2 {
+    grid-column: span 2 / span 2;
+}
+
+.setup,
+.mapping {
+    composes: no-scrollbar;
+}
+
+.setup .logo {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    min-height: 200px;
+    background-color: var(--framer-color-bg-secondary);
+    border-radius: 10px;
+}
+
+.setup label {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 30px;
+    width: 100%;
+    justify-content: space-between;
+    padding-left: 15px;
+    color: var(--framer-color-text-tertiary);
+}
+
+.setup select {
+    width: 164px;
+    text-transform: capitalize;
+}
+
+.mapping {
+    padding-bottom: 0;
+}
+
+.mapping .fields {
+    display: grid;
+    grid-template-columns: 1fr 8px 1fr;
+    gap: 10px;
+    margin-bottom: auto;
+    padding-bottom: 10px;
+    align-items: center;
+    color: var(--framer-color-text-tertiary);
+}
+
+.mapping .slug-field {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: space-between;
+    gap: 10px;
+    color: var(--framer-color-text-tertiary);
+}
+
+.mapping .source-field {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    white-space: nowrap;
+    height: 30px;
+    padding: 0px 8px;
+    font-size: 12px;
+    color: var(--framer-color-text);
+    background-color: var(--framer-color-bg-tertiary);
+    border-radius: 8px;
+    cursor: pointer;
+    gap: 8px;
+    user-select: none;
+}
+
+.mapping .source-field[aria-disabled="true"] {
+    opacity: 0.5;
+}
+
+.mapping .source-field:focus-within {
+    box-shadow: inset 0 0 0 1px var(--framer-color-tint);
+}
+
+.mapping .source-field input[type="checkbox"] {
+    cursor: pointer;
+}
+
+.mapping .source-field input[type="checkbox"]:focus {
+    box-shadow: none;
+}
+
+.mapping footer {
+    position: sticky;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    background-color: var(--framer-color-bg);
+    margin-top: auto;
+    padding-bottom: 15px;
+    display: flex;
+    flex-direction: column;
+    gap: 15px;
+}
+
+.mapping footer::before {
+    content: "";
+    position: absolute;
+    top: -45px;
+    left: 0;
+    width: 100%;
+    height: 45px;
+    background: linear-gradient(to bottom, transparent, var(--framer-color-bg));
+    pointer-events: none;
+}
```

Comment:
not in use

Reply:
oh no, this is the one in use not the `.scroll-fade`